rsts/concepts/flyte_console.rst

```diff
@@ -0,0 +1,231 @@
+.. _ui:
+
+How to Use Flyte UI
+===================
+
+Flyte UI is a web-based user interface for Flyte. It helps interact with Flyte objects and builds DAGs out of your workflows.
+
+With Flyte UI, you can:
+
+* Launch Workflow
+* Launch Task
+* View Versioned Tasks and Workflows
+* Trigger Versioned Tasks and Workflows
+* Inspect Execution through Inputs, Outputs, Logs, and Graphs
+* Clone Execution
+* Relaunch Execution
+* Recover Execution
+
+.. note::
+    `Flyte Console <https://github.com/flyteorg/flyteconsole>`__ hosts the Flyte user interface code.
+
+Launch Workflow
+---------------
+
+Launch a workflow by clicking on the **Launch Workflow** button. Workflows are viewable after they are registered.
+UI should be accessible at http://localhost:8000/console.
```

Comment:
Where is this port coming from? if you run `flytectl sandbox start` and that's what we are documenting, then we should use the following as the url:
```suggestion
UI should be accessible at http://localhost:30081/console.
```

I see a few instances of this through out the document

Reply:
@EngHabu: Done. Added to the current branch.

Reply:
So sorry. My bad. :/